crates/nu-command/src/platform/ansi/link.rs

```diff
@@ -0,0 +1,162 @@
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::{Call, CellPath},
+    engine::Command,
+    engine::EngineState,
+    engine::Stack,
+    Category, Example, PipelineData, ShellError, Signature, Span, Spanned, SyntaxShape, Type,
+    Value,
+};
+
+#[derive(Clone)]
+pub struct SubCommand;
+
+impl Command for SubCommand {
+    fn name(&self) -> &str {
+        "ansi link"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("ansi link")
+            .input_output_types(vec![
+                (Type::String, Type::String),
+                (
+                    Type::List(Box::new(Type::String)),
+                    Type::List(Box::new(Type::String)),
+                ),
+                (Type::Table(vec![]), Type::Table(vec![])),
+                (Type::Record(vec![]), Type::Record(vec![])),
+            ])
+            .named(
+                "text",
```

Comment:
```suggestion
                "title",
```
It just seems to me that `--title` is the better name for this parameter than `--text`. What do you think?

Reply:
I don't think so. For example in HTML title attribute has meaning of tooltip. For example:
```html
<a href="https://nushell.sh" title="A new type of shell">Nushell</a>
```
generates

![image](https://user-images.githubusercontent.com/17511668/212531211-ceef2828-fd5c-46b5-8d0e-fcdaaad873ec.png)

I think naming parameter titile may be confusing

Reply:
That's the point I'm making. In your html example, the `title` is "A new type of shell" so the link shows "A new type of shell" instead of showing "https://nushell.sh". That's why I'm suggesting it.

Reply:
Exactly. But the `--text` flag controls text of link, not tooltip. This is a call equivalent to my html example:

![image](https://user-images.githubusercontent.com/17511668/212545103-c00d0f80-c65b-479e-aa31-7ddd5b8f5810.png)

And as far as i understand there is no way to alter tooltip. In my screenshot it is generated by windows terminal and just shows what content of link is hidden

![image](https://user-images.githubusercontent.com/17511668/212545158-01fb7ddd-2b7d-4178-ac71-b9e2d9fa18f6.png)

Reply:
I know of no way to control the tooltip either, but I wasn't talking about the tool tip. I prefer title but I'm not going to die on this hill. It's not that big of a deal. I'll let someone else accept or deny this PR.